java/src/jmri/jmrit/dispatcher/AutoActiveTrain.java

```diff
@@ -621,6 +621,9 @@ else if (_activeTrain.isTransitReversed() && as.getSequence() == _activeTrain.ge
                         // set the blocks as normal
                         _previousBlock = _currentBlock;
                         _nextBlock = getNextBlock(b, as);
+                        //if (_nextBlock.getState() == Block.OCCUPIED) {
+                        //    handleBlockStateChange(as, _nextBlock);
+                        //}
```

Comment:
Just checking. These lines are added but commented out.

Reply:
That is correct. Will/maybe happen later.